api_v3/lib/types/KalturaUiConfV2Redirect.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * @package api
+ * @subpackage objects
+ */
+
+class KalturaUiConfV2Redirect extends KalturaObject
+{
+	/**
+	 * @var int
+	 */
+	public $v7id;
+
+	/**
+	 * @var bool
+	 */
+	public $isApproved;
+
+	/**
+	 * @var bool
+	 */
+	public $translatePlugins;
+
+	private static $map_between_objects = array
+	(
+		"v7id" ,
+		"isApproved" ,
+		"translatePlugins"
+	);
+
+	public function getMapBetweenObjects ( )
+	{
+		return array_merge ( parent::getMapBetweenObjects() , self::$map_between_objects );
+	}
+
+	public function toObject($object_to_fill = null, $props_to_skip = array())
+	{
+		$this->validatePropertyNumeric('v7id');
+		$this->validatePropertyNotNull('isApproved');
+		$this->validatePropertyNotNull('translatePlugins');
```

Comment:
Why is this mandatory and not false by default?

Reply:
to avoid confusion, if this object is touched, all his values must be set.